test/rules/base.jl

```diff
@@ -1,10 +1,31 @@
-function test_scalar(f, f′, xs...)
+_isapprox(x, y; kwargs...) = isapprox(x, y; kwargs...)
+
+const AbstractDifferential = ChainRules.AbstractDifferential
```

Comment:
This is not how you import specific bindings in Julia. Use `using ChainRules: AbstractDifferential` at the top of `test/runtests.jl` if the binding you want is not already exported.

Reply:
Will do. The problem was that having `using ChainRules: AbstractDifferential` in this file, made the test crash on my machine, for some reason, so this seemed like the quickest fix.

Reply:
Hm, that's... bizarre.